src/simpleDetFunc.c

```diff
@@ -130,15 +130,15 @@ static inline void remove_var(double val, int *nobs, double *mean_x,
 }
 
 /* Basically we compute the rolling variance following the algorithm from 
- * python pandas. (This algorithm can still be further optimized to be much 
- * more similar to the standard deviation algorithm used for the alternative 
- * detection function calculation. (It just requires a more careful 
- * implementation that I don't currently have the patience for).
+ * python pandas. This algorithm can still be further optimized so that we add
+ * and remove values from the windows in chunks (rather than 1 at a time). 
+ * Additionally, if we store wInd on the stack, we might get better 
+ * optimizations.
  */
 
-void rollSigma(int startIndex, int interval, float scaleFactor,
-	       int sigWindowSize, int dataLength, int numWindows,
-	       float *buffer, float *sigmas)
+int rollSigma(int startIndex, int interval, float scaleFactor,
```

Comment:
Same as centralFreqMapper, why was this changed to return an int instead of void?